docs/t-sql/language-elements/declare-cursor-transact-sql.md

```diff
@@ -151,12 +151,15 @@ Specifies that positioned updates or deletes made through the cursor are guarant
 
 #### OPTIMISTIC
 
-Specifies that positioned updates or deletes made through the cursor don't succeed, if the row was updated since it was read into the cursor. [!INCLUDE [ssNoVersion](../../includes/ssnoversion-md.md)] doesn't lock rows as they are read into the cursor. It instead uses comparisons of **timestamp** column values, or a checksum value if the table has no **timestamp** column, to determine whether the row was modified after it was read into the cursor. If the row was modified, the attempted positioned update or delete fails. `OPTIMISTIC` can't be specified if `FAST_FORWARD` is also specified.
+Specifies that positioned updates or deletes made through the cursor don't succeed, if the row was updated since it was read into the cursor. [!INCLUDE [ssNoVersion](../../includes/ssnoversion-md.md)] doesn't lock rows as they are read into the cursor. It instead uses comparisons of **timestamp** column values, or a checksum value if the table has no **timestamp** column, to determine whether the row was modified after it was read into the cursor. If the row was modified, the attempted positioned update or delete fails. `OPTIMISTIC` can't be specified if `FAST_FORWARD` is also specified, and is ignored if `STATIC` is also specified.
 
 #### TYPE_WARNING
 
 Specifies that a warning message is sent to the client when the cursor is implicitly converted from the requested type to another.
 
+> [!NOTE]  
```

Comment:
I would also suggest the following note:
No warning is sent to the client when the combination of `OPTIMISTIC` and `STATIC` cursor properties are used, and the cursor is implicitly converted to the equivalent of a `STATIC` `READ_ONLY` or `STATIC` `FORWARD_ONLY` cursor.

The conversion to READ_ONLY turns into a FAST_FORWARD + READ_ONLY cursor from a clients' perspective. Where a client would see what the sys.dm_exec_cursors DMV returns.